service/controller/internal/cloudconfig/cloud_config_test.go

```diff
@@ -49,9 +49,20 @@ func Test_Service_CloudConfig_NewMasterTemplate(t *testing.T) {
 			t.Fatalf("expected %#v got %#v", nil, err)
 		}
 
+		versions := k8scloudconfig.Versions{
+			Calico:                       "3.14.1",
+			CRITools:                     "0.1.0",
+			Etcd:                         "3.4.9",
+			Kubernetes:                   "1.16.12",
+			KubernetesAPIHealthz:         "0.1.1",
+			KubernetesNetworkSetupDocker: "0.2.0",
+		}
```

Comment:
Just curious - is there interest / do you think it's worth it to get these from releases in legacy?

Reply:
These are just test values--we already implemented "versions from releases" in this operator.